Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -914,9 +905,10 @@ class KalmanFitter {
   /// Fit implementation of the foward filter, calls the
   /// the forward filter and backward smoother
   ///
-  /// @tparam source_link_t Source link type identifying uncalibrated input
-  /// measurements.
+  /// @tparam source_link_t Type of the source link
```

Comment:
Cosmetic change

Reply:
Exactly, that is the point. Says the same thing and fits on the same line.